lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -10,7 +10,7 @@
             'text!../../../templates/financial-assistance/financial_assessment_form.underscore',
             'text!../../../templates/financial-assistance/financial_assessment_submitted.underscore',
             'text!templates/student_account/form_field.underscore',
-            'string_utils'
+            'edx-ui-toolkit/js/utils/html-utils'
```

Comment:
Nit: I prefer putting ``HtmlUtils`` earlier in the list, probably after ``gettext``. I feel that the dependencies should be the generic helpers first, and then the more specific libraries later.